src/main/java/baseball/Validation.java

```diff
@@ -0,0 +1,62 @@
+package baseball;
+
+import java.util.HashSet;
+
+import static utils.Constant.*;
+
+public class Validation {
+    public boolean validateInputNumber(String input) {
+
+        if (!isRightDigit(input)) {
+            return false;
+        }
+
+        if (!isNumber(input)) {
+            return false;
+        }
+
+        if (isDuplicate(input)) {
+            return false;
+        }
+
+        return true;
+    }
+
+    public boolean validateContinueNumber(String input) {
+        if (input.equals(RESTART) || input.equals(EXIT)) {
+            return true;
+        } else {
+            return false;
+        }
+    }
+
+    public boolean isRightDigit(String input) {
+        if (input.length() != DIGIT) {
+            return false;
+        } else {
+            return true;
+        }
+    }
+
+    public boolean isNumber(String input) {
+        for (char c : input.toCharArray()) {
+            if (c < '0' || c > '9') {
+                return false;
+            }
+        }
+        return true;
+    }
+
+    public boolean isDuplicate(String input) {
+        HashSet<Character> duplicateCheck = new HashSet<>();
+
+        for (char c : input.toCharArray()) {
+            if (!duplicateCheck.contains(c)){
+                duplicateCheck.add(c);
+            } else {
+                return true;
+            }
+        }
+        return false;
```

Comment:
좀 더 간결하게 표현할 수 있을 것 같아요 :) 라인 수를 줄이는 것이 절대적인 미덕은 아니지만, 조금 더 간결하게 표현할 수 있다면 가독성이 향상하기 때문에 염두에 두면서 코딩하는 것이 좋다고 생각합니다

```suggestion
        for (char c : input.toCharArray()) {
                duplicateCheck.add(c);
        }
        return duplicateCheck.size() != input.length();
```

혹은 스트림에 대해서 학습해보시면, 더 간결하고 명시적으로 표현할 수 있습니다~~!
```suggestion
        long count = input.chars()
            .distinct()
            .count();

        return count != input.length();
```

Reply:
와, 제안해주신 코드 보니까 훨씬 간단하네요!
1. 바로 리턴해버리는 것을 염두했었는데, 최대한 이 코드의 리턴이 무엇을 의미하는지 네이밍을 통해 보여주는 것이 협업 시 좋을 것 같다는 생각에 최대한 나열해서 작성했던 것 같아요.
`return function()`
`int result = function() return result` 이런 식으로 풀어서 바로 리턴해보겠습니다!!

2. 스트림은 보기에도 직관적이고 간결해서 바로 공부하러 가보겠습니다 ㅎㅎ